packages/markdown/prepareMarkdown.js

```diff
@@ -102,9 +103,8 @@ function prepareMarkdown(config) {
         contents.push(`
 ## Unstyled
 
-:::success
-[Base UI](/base-ui/) provides a headless ("unstyled") version of this [${title}](${headers.unstyled}). Try it if you need more flexibility in customization and a smaller bundle size.
-:::
+A version of this component is available without any styles: [Base UI ${markdownH1}](${headers.unstyled}).
```

Comment:
```suggestion
This Material UI ${markdownH1} component is built on top of [Base UI ${markdownH1}](${headers.unstyled}), a similar component but without any default styles.
```

Reply:
Is that true already for all of the pages where this callout appears? The current copywriting just says that there's a Base UI version of the same component.

Reply:
Oh, true, I forgot it was not true yet 🙃

Reply:
```suggestion
To completely own the design, without having to customize parting from Material Design, use the [Base UI ${markdownH1}](${headers.unstyled}) for an unstyled version of this component
```

I'd probably do something like this as it goes in the direction that Sam was pushing a bit more: we're explaining _why_; the value of picking the Base UI version of the component. In this case, it'd be "to completely own the design" and "to customize without having to deal with Material Design".